lms/static/js/edxnotes/views/tabs/tags.js

```diff
@@ -0,0 +1,117 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'gettext', 'jquery', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
+    'js/edxnotes/views/tab_view'
+], function (gettext, $, _, NoteGroupView, TabPanelView, TabView) {
+
+    var TagsView = TabView.extend({
+        scrollToTag: function(tagName) {
+            var titleElement;
+            if (!this.tabModel.isActive()) {
+                this.tabModel.activate();
+            }
+            titleElement = this.$el.find('.tags-title').filter(
+                // Since tags can only be single words, we are safe just looking for the title
```

Comment:
I don't understand the advantage of this, and it complicates the code. What is the risk of comparing text if if I know exactly what the text is? Note that I am scoping to the class "tags-title".